luatest/assertions.lua

```diff
@@ -127,14 +127,14 @@ end
 -- @string message
 function M.skip_if(condition, message)
     if condition and condition ~= nil then
-        utils.luatest_error('skip', message, 2)
+        utils.luatest_error('skip', message or 'No reason', 2)
```

Comment:
Hey, but we know the reason here. It's because `condition` was true. 

```suggestion
        utils.luatest_error('skip', message or '(skip condition was true, no message specified)', 2)
```

Reply:
IMHO, it's redundant. Let's have just `(no reason specified)` which is more than enough.